xarray/core/dataset.py

```diff
@@ -2008,6 +2008,90 @@ def sel(
         result = self.isel(indexers=pos_indexers, drop=drop)
         return result._overwrite_indexes(new_indexes)
 
+    def head(
+        self, indexers: Mapping[Hashable, Any] = None, **indexers_kwargs: Any
+    ) -> "Dataset":
+        """Returns a new dataset with the first `n` values of each array
+        for the specified dimension(s).
+
+        Parameters
+        ----------
+        indexers : dict, optional
+            A dict with keys matching dimensions and integer values `n`.
+            One of indexers or indexers_kwargs must be provided.
+        **indexers_kwarg : {dim: n, ...}, optional
```

Comment:
```suggestion
        **indexers_kwargs : {dim: n, ...}, optional
```

Reply:
Thanks for the correction! I found multiple places where it's written in singular form. I'll see what else I can find to contribute to the docs.